[PL-42246] : Added more information in docs to have better clarity (#766)
harness/terraform-provider-harness: docs/resources/platform_resource_group.md

- `allowed_scope_levels` (Set of String) The scope levels at which this resource group can be used
- `color` (String) Color of the environment.
- `description` (String) Description of the resource.

> - `included_scopes` (Block Set) Included scopes; default chosen based on resource group scope if not specified. (see [below for nested schema](#nestedblock--included_scopes))

Reviewer: ```suggestion
- `included_scopes` (Block Set) Included scopes. The default is selected based on the resource group scope if not specified. (Go to [nested schema](#nestedblock--included_scopes) below.)
```
